.github/workflows/build-with-bal-test-native.yml

```
@@ -0,0 +1,29 @@
+name: Build with bal test native
+
+on:
+    workflow_dispatch:
+        inputs:
+            lang_tag:
+                description: Branch/Release Tag of the Ballerina Lang
+                required: true
+                default: master
+            lang_version:
+                description: Ballerina Lang Version (If given ballerina lang buid will be skipped)
+                required: false
+                default: ''
+            native_image_options:
+                description: Default native-image options
+                required: false
+                default: ''
+    schedule:
+        -   cron: '30 18 * * *'
+
+jobs:
+    call_stdlib_workflow:
+        name: Run StdLib Workflow
+        if: ${{ github.event_name != 'schedule' || (github.event_name == 'schedule' && github.repository_owner == 'ballerina-platform') }}
+        uses: ballerina-platform/ballerina-standard-library/.github/workflows/build-with-bal-test-native-template.yml@main
+        with:
+            lang_tag: ${{ inputs.lang_tag }}
+            lang_version: ${{ inputs.lang_version }}
+            native_image_options: ${{ inputs.native_image_options }}
```

Need to add this additional flag to disable some tests on windows. (See the [pull request workflow](https://github.com/ballerina-platform/module-ballerina-jwt/blob/eee083bf9d8f016b9f9dacfbc3254e0651327ed7/.github/workflows/pull-request.yml#L39))
```suggestion
            native_image_options: ${{ inputs.native_image_options }}
            additional_windows_build_flags: '-Pdisable=jwks'
```